### PR TITLE
Changes the punishment for calling the maids for help to a timer padlock

### PR DESCRIPTION
**Pull request by Ada18980**
Also removes the wardrobe blocking since many players aren't comfortable with that.

### CODE REVIEW

File: BondageClub/Screens/Room/MainHall/MainHall.js [hunk 1]
```diff
@@ -491,7 +491,7 @@ function MainHallPunishFromChatroom() {
 function MainHallPunishFromChatroomStartPunishment() {
 	CharacterRelease(Player);
 	CharacterNaked(Player);
-	LogAdd("BlockChange","Rule", CurrentTime + 3600000);
+	//LogAdd("BlockChange","Rule", CurrentTime + 3600000);
```
Comment (elliesec) [inline BondageClub/Screens/Room/MainHall/MainHall.js:494]: If you're removing the wardrobe block, might as well remove the line completely.

File: BondageClub/Screens/Room/MainHall/MainHall.js [hunk 2]
```diff
@@ -491,7 +491,7 @@ function MainHallPunishFromChatroom() {
...
 	
 	// Apply one of several preset restraints
 	// Also  apply mistress locks to everything
```
Comment (elliesec) [inline BondageClub/Screens/Room/MainHall/MainHall.js:497]: Probably want to change this comment now to reflect the changes.

**Comment by Ada18980** [comment]
Thanks @elliesec !

**Comment by Ben987** [comment]
Sure, this is a little less harsher.  Thanks!

**Pull request merged**